packages/walletconnect/package.json

```
@@ -15,6 +15,7 @@
   "devDependencies": {
     "@sovryn/onboard-eslint-config": "*",
     "@sovryn/onboard-tsconfig": "*",
+    "@walletconnect/types": "^2.8.1",
```

```suggestion
    "@walletconnect/types": "2.8.1",
```
How do you feel about setting static versions for the new packages?